asm2vec/tensors.py

```diff
@@ -0,0 +1,69 @@
+import os
+import torch
+import logging
+import asm2vec
```

Comment:
You shouldn't have this open import for the package your working inside... will create circular dependencies